src/components/menu-top/MenuTop.vue

```diff
@@ -20,6 +20,13 @@
 
           <hr class="navbar-divider">
 
+          <b-navbar-item
+            data-test="menu-top--import-calchart3"
```

Comment:
Let's add a unit and e2e test to make sure this opens the modals
```suggestion
            data-test="menu-top--load-show"
```

Reply:
Added!